config.local.php.example

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$rsaKeys = explode(',', file_get_contents('.rsa_keys'));
+
+return [
+    'telegram' => [
+        'authorization' => [
+            'rsa_keys' => $rsaKeys,
+        ],
+        'connection' => [
+            'main' => [
+                'ipv4' => [
+                    // ipv4 addresses
+                    2 => [
+                        // The rest will be fetched automatically
+                        'ip_address' => '149.154.167.50',
+                    ],
+                ],
+            ]
+        ]
+    ],
+];
```

Comment:
Are you sure you need to change rsa keys and servers of telgram. @danog always said its not necessary. 

But custom config would be convinient. We just need custom method for correct merging of configs. Maybe there is such method in madelineProto itself.